application/asset/js/resource-form.js

```diff
@@ -246,6 +246,7 @@
         }
         var value = $('.value.template[data-data-type="' + dataType + '"]').clone(true);
         value.removeClass('template');
+        value.attr('data-term', term);
```

Comment:
Is there a reason some of these are using `.attr('data-term' ...` instead of `.data('term', ...` ?

Reply:
I didn't checked all the cases, but in https://github.com/omeka/omeka-s/blob/develop/application/asset/js/resource-form.js#L340, you indicated that there may be some issues without attr. Here, it is useless, so data can be used. But the data is required (else the module cannot determine the property of the value to be created).

Reply:
Okay, you're referring to an existing comment, I see.

I believe that what that comment refers to is other parts of the code using attribute selectors to find the elements: when that functionality is being used, then we need to use `attr`, or else there's no attribute to select against.

Reply:
Ok, I convert it into data.